docs/RELEASE_TEST_SCRIPT.md

```diff
@@ -0,0 +1,17 @@
+This document should contain plain english instructions for testing functionality on `gaiad`. This “Script” is supposed to be run by 2 people who will each spin up a `gaiad` node and run the series of prompts below.
+
+- Create a network of 2 nodes
+- Generate an account
+- Send funds from one account to the other
+- Create a validator
+- Edit a validator
+- Delegate to validator
+- Unbond from a validator
+- View validators and verify output
+- Query network status
+- Create a proposal
+- Query a proposal
+- Vote on a proposal
+- Query status of a proposal
+- Query the votes on a proposal
+- Export state
```

Comment:
What's the use of doing this without re-importing state? Just to test that `gaiad export` doesn't crash?

Reply:
yeah also funny things could happen on export state (and reloading from an export) like information not being carried properly - maybe we should check that

Reply:
^ but yeah export state didn't work previously - it was just broken on master for a while

Reply:
This was suggested in the meeting yesterday so I added it.

Reply:
I don't think so - takes 5 seconds anyways

Reply:
Can we try "export and reload"? I think that's way more informative, and it only takes 10 seconds instead of 5!

(all you need to do is `gaiad export > genesis.json; cp genesis.json ~/.gaiad/config/genesis.json; gaiad start`)

Reply:
cc @zramsay do you mind adding that while you are making changes here?